api-reference/openapi.yaml

```diff
@@ -44,42 +44,94 @@ components:
   schemas:
     Body__partition_pdf_async_v1_document_partition_post:
       properties:
-        pdf:
+        file:
           type: string
-          format: binary
-          title: Pdf
+          format: base64
+          title: file
         options:
-          anyOf:
-            - type: string
-              format: binary
-            - type: 'null'
-          title: Options
+          type: object
+          # additionalProperties: true
+          properties:
+            selected_pages:
+              type: array
+              items:
+                oneOf:
+                  - type: integer
+                  - type: array
+                    items:
+                      type: integer
+              minItems: 2
+              maxItems: 2
```

Comment:
This is not indented properly. The min/max should apply to the innermost array.

Reply:
Ah good catch, I was doing some experimentation. Fixed.